plugin/pick.vim

```diff
@@ -68,3 +72,15 @@ function! s:BufferListCommand()
   let buffers = map(bufnrs, 'bufname(v:val)')
   return 'echo "' . join(buffers, "\n") . '"'
 endfunction
+
+function! s:TagCommand()
+  let l:tag_files = []
+
+  for tfile in split(&tags, ",")
+    if filereadable(expand(tfile)) == 1
```

Comment:
Testing file readability is susceptible to race conditions.

Could this be avoided by using cat to to join the existing files before invoking awk?

```viml
function! s:TagCommand()
  return "cat " . join(split(&tags, ","), " ") . " 2> /dev/null | awk -F$'\t' '{print $1}'  | sort -u | grep -v '^!'"
endfunction
```